### PR TITLE
Update sample project for gremlin

### PR DESCRIPTION
**Pull request by chenrujun**

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , @JimSuplizio .
Please help to review. 🙏

**Comment by chenrujun** [comment]
Hi, @mitchdenny  , Please help to review. 🙏

File: sdk/spring/azure-spring-boot-samples/azure-spring-data-sample-gremlin/pom.xml [hunk 1]
```diff
@@ -1,46 +1,55 @@
...
+		<relativePath/>
+	</parent>
+	<groupId>com.azure</groupId>
+	<artifactId>azure-spring-data-sample-gremlin</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>azure-spring-data-sample-gremlin</name>
+	<description>Sample project for cosmos gremlin</description>
+
+	<properties>
+		<java.version>1.8</java.version>
```
Comment (JimSuplizio) [inline sdk/spring/azure-spring-boot-samples/azure-spring-data-sample-gremlin/pom.xml:18]: Why is this necessary?

**Comment by chenrujun** [comment]
@JimSuplizio .
Thank you for your review.
I have updated this PR according to your suggestion.
Please check. 🙏
